homeassistant/components/homekit/accessories.py

```diff
@@ -572,21 +605,30 @@ def async_call_service(
         )
 
     @ha_callback
-    def async_reset(self) -> None:
+    def async_reload(self) -> None:
         """Reset and recreate an accessory."""
```

Comment:
Should this docstring be updated?

Reply:
d3b7d8e3675f933e4be7f986503b050d4a81b286